.gitignore

```diff
@@ -24,4 +24,6 @@ spring-xd-samples/*/xd
 dump.rdb
 coverage-error.log
 .apt_generated
-aws.credentials.properties
\ No newline at end of file
+aws.credentials.properties
+/spring-cloud-stream-binder-kinesis/nbproject/
+nb-configuration.xml
```

Comment:
What are these two entries?
Why do we need to exclude globally for all the contributors?

Reply:
These are IDE specific files. I think they are useful in a way that you make sure these files are not committed by mistake. I can add a comment or get rid of them as you prefer.

Reply:
Ah! NetBeans. I see the first time its specific files.
OK. Let it be then!